test/easyconfigs/easyconfigs.py

```diff
@@ -653,6 +659,11 @@ def check_https(self, changed_ecs):
 
     def test_changed_files_pull_request(self):
         """Specific checks only done for the (easyconfig) files that were changed in a pull request."""
+        def get_eb_files_from_diff(filter):
```

Comment:
@Flamefire `filter` is a Python keyword (see syntax highlighting in the diff), so let's use `filter_char` instead?

Reply:
I used diff_filter as per the git command